week4/solutions/scrabble/tests/test_score.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+import unittest
+import unittest.mock
+
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir))
+
+from score import Score
+from word import Word
+
+#Score.get_total()
+#Score.get_round()
+#Score.set_total()
+#Score.set_round()
+#Score.reset()
+class ScoreTest2(unittest.TestCase):
```

Comment:
Should I separate these out into a few more tests? Or is it okay to lump them together like this? My instinct would be that separation would make it more clear when they fail what the issue is which might be better.

Reply:
It's totally cool to lump them together, usually logically.